.github/workflows/inspektor-gadget.yml

```diff
@@ -65,26 +72,29 @@ jobs:
 
     - uses: azure/docker-login@v1
       with:
-        username: ${{ secrets.DOCKER_USERNAME }}
-        password: ${{ secrets.DOCKER_PASSWORD }}
+        username: ${{ secrets.CONTAINER_REGISTRY_USERNAME }}
+        password: ${{ secrets.CONTAINER_REGISTRY_PASSWORD }}
 
     - name: Integration tests
+      env:
+        CONTAINER_REPO: ${{ secrets.CONTAINER_REPO }}
+        IMAGE_TAG: ${{ steps.publish-registry.outputs.snapshot-tag }}
       run: |
-        export IMAGE_TAG=${{steps.publish-registry.outputs.snapshot-tag}}
         echo "Using IMAGE_TAG=$IMAGE_TAG"
 
-        make -C integration build push
-
         TESTS_DOCKER_ARGS="-e KUBECONFIG=/root/.kube/config -v /home/runner/.kube:/root/.kube -v /home/runner/work/_temp/.minikube:/home/runner/work/_temp/.minikube" \
-            make -C integration test
+            make -C integration build test
```

Comment:
We need to take care whether `make -C integration build` builds the container image in docker on the host or in minikube.

If it is meant to be built on the host, the Kubernetes cluster in minikube will not have access to the container image unless it is pushed to the registry.

If it is meant to be built in minikube, I guess you need to add the following either here or in the Makefile:
```
`eval $(shell $(MINIKUBE) docker-env | grep =)`
```
(there is an example of that in `gadget-container/Makefile`).

And then, change `imagePullPolicy: Always` to `Never` in `integration/gadget-integration-tests-job.yaml`.

Reply:
I'm not getting it. We are not deploying `integration/gadget-integration-tests-job.yaml` here, we are just using docker and passing the minikube kubeconfig to it. The Github actions appear to be working fine at this point:
- On this repo : https://github.com/kinvolk/inspektor-gadget/runs/762231976?check_suite_focus=true
- On my personal fork: https://github.com/mauriciovasquezbernal/inspektor-gadget/runs/762218463?check_suite_focus=true

Reply:
You're right. Sorry, I got things mixed up.